### ISSUE TITLE
Paste/convert rich text.

### ISSUE THREAD

**Issue by dgk4201** [fabiocolacio/Marker#226]
I'm trying to get off MacOS. I use Ulysses a lot. One of the features I need is the ability to paste rich text into the editor and have it converted to MD. Any chance we'll see this feature?

**Comment by fabiocolacio** [comment]
Good idea! I am planning a re-write of Marker (or at least a major refactor) to make it easier to maintain, so there will be a feature-freeze for a while during that process, but this will certainly be added to the newer version.
